milabench/config.py

```diff
@@ -73,3 +73,29 @@ def build_config(*config_files):
     for name, bench_config in all_configs.items():
         all_configs[name] = finalize_config(name, bench_config)
     return all_configs
+
+
+def build_system_config(config_file):
+    config_file = XPath(config_file).absolute()
```

Comment:
if the system path does not exist we could create a default "single node" system config so we can avoid having `if system` 

something with those defaults
```
system.arch = deduce_arch()
system.sshkey= None
system.nodes: [
    { 
        name: local, 
        ip: 127.0.0.1, 
        port: NA, 
        user: os.getlogin(), 
        main_node: True 
    }
]
```